docs/design/Education-organization-Endpoints.md

```diff
@@ -0,0 +1,153 @@
+# Education Organizations Endpoints
+
+## Overview
+
+Provides a consolidated view of education organizations across all Ed-Fi ODS
+instances through REST API endpoints. The data is refreshed on a scheduled
+basis.
+
+## Features
+
+* **REST API Endpoints:**
+  * `GET /v2/educationorganizations` or `GET /v1/educationorganizations` -
+    Returns all education organizations from all instances
+  * `GET /v2/educationorganizations/{instanceId}` or
+    `GET /v1/educationorganizations/{instanceId}` - Returns education organizations
+    for a specific instance
+
+* **Automatic Data Refresh:**
+  * Quartz.NET scheduled job runs every 6 hours by default
+  * No manual refresh endpoint - data is refreshed automatically via background
+    job
```

Comment:
We would likely need the ability to trigger a synchronization. If someone adds an ed org, they may want to get that synchronized right away.